src/pages/Support.vue

```diff
@@ -36,6 +36,16 @@ export default {
     },
     mdScreen() {
       return this.$vuetify?.breakpoint?.mdAndUp
+    },
+    messageWithLogRocketSession() {
+      let url = ''
+      LogRocket.getSessionURL(sessionURL => {
+        url = sessionURL
+      })
+      return this.selectedCategory === 'I need help' ||
+        this.selectedCategory === "I've found a bug"
+        ? `${this.message} Relevant LogRocket URL: ${url}`
+        : this.message
```

Comment:
I think we should wrap this in a try catch in case the LogRocket instance fails for some reason - we don't want to block submission of the form in those cases.

Reply:
Thanks -I'll switch to draft and update next week